### PR TITLE
Spectrum latest stylesheets

### PR DESCRIPTION
**Pull request by nGolubeva**

### CODE REVIEW

**Comment by jenstroeger** [comment]
@nGolubeva, the files in `extensions/Bookalope/css/spectrum/` are from the **Resources** on the [Adobe Spectrum](https://spectrum.adobe.com/#Resources) page?

**Comment by nGolubeva** [comment]
> @nGolubeva, the files in `extensions/Bookalope/css/spectrum/` are from the **Resources** on the [Adobe Spectrum](https://spectrum.adobe.com/#Resources) page?

@jenstroeger, yes, these are the Adobe Spectrum stylesheets

**Review by jenstroeger — DISMISSED** [review]

File: extensions/Bookalope/css/bookalope-dark.css [hunk 1]
```diff
@@ -1,5 +1,9 @@
 @charset "utf-8";
 
+.spectrum--dark {
+    --spectrum-fieldlabel-m-asterisk-color: inherit;
+}
```
Comment (jenstroeger) [inline extensions/Bookalope/css/bookalope-dark.css:5]: This is a class `.spectrum-dark` in the `bookalope-dark.css` file. Next…

File: extensions/Bookalope/css/bookalope-darkest.css [hunk 2]
```diff
@@ -1,5 +1,9 @@
 @charset "utf-8";
 
+.spectrum--dark {
+    --spectrum-fieldlabel-m-asterisk-color: inherit;
+}
```
Comment (jenstroeger) [inline extensions/Bookalope/css/bookalope-darkest.css]: This is a class `.spectrum-dark` in the `bookalope-darkest.css` file — not called `.spectrum-darkest`? And there isn’t such a class in the two light/lightest css files? So:
```css
.spectrum--darkest { ... }
```

**Review by jenstroeger — COMMENTED** [review]
But it seems to work. Just curious, because it feels inconsistent to me? I would have expected a rhythm… light css file/light css class; dark css file/dark css class/etc. But I may be wrong.

**Comment by jenstroeger** [comment]
> But it seems to work. Just curious, because it feels inconsistent to me? I would have expected a rhythm… light css file/light css class; dark css file/dark css class/etc. But I may be wrong.

Answer from @nGolubeva:

> no, this change updates the color for the asterisk. For light themes, the default color looks good (https://prnt.sc/12h29ky). For dark themes, I've changed the color of the asterisk.

File: extensions/Bookalope/css/bookalope-darkest.css (hunk 2 above)
Comment (nGolubeva) [reply extensions/Bookalope/css/bookalope-darkest.css]: I've fixed the class name for `bookalope-darkest.css`.
No, there is no such class for light/lightest themes.

**Review by jenstroeger — APPROVED** [review]

**Pull request merged**